Content.Server/_NF/GameRule/NfAdventureRuleSystem.cs

```diff
@@ -150,6 +151,17 @@ private void OnStartup(RoundStartingEvent ev)
             _shuttle.SetIFFColor(depotUid3s[0], depotColor);
         }
 
+        if (_map.TryLoad(mapId, nfsdStation, out var nfsdUids, new MapLoadOptions
+            {
+                Offset = _random.NextVector2(500f, 700f)
+            }))
+        {
+            // We should figure out if it is possible to add this grid to the latejoin listing.
```

Comment:
what you'll need to do is give it a station prototype, give the grid the matching BecomesStation component info, and then initialize it as a station here after its been spawned. See the Exped Lodge for how i've done it there.

Reply:
Thanks for the pointer! I'll get to work on this in a bit.